OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -224,6 +224,8 @@ public override void MouseExited()
 			tooltipContainer.Value.RemoveTooltip();
 		}
 
+		public override string GetCursor(int2 pos) { return ChromeMetrics.Get<string>("ButtonCursor"); }
```

Comment:
Can we please define a `public string Cursor = ChromeMetrics.Get<string>("ButtonCursor");` and change this to `return Cursor`?

This will allow mods to use different cursors for different button types, like they already can for colour/clicksound/visual height.

Reply:
Okay, why not.